lib/crypto/src/elliptic-curve/test_vectors/field.rs

```diff
@@ -0,0 +1,255 @@
+//! Test vectors for the secp256r1 base field.
```

Comment:
How is it generated? Any tool? Trying to figure out edge cases?

Reply:
So, these are just the repeated doubling of the multiplicative identity. That is, `1, 2, 4, 8, 16, 32`, so basically the powers of two.